src/main/scala/com/johnsnowlabs/nlp/util/io/ResourceHelper.scala

```diff
@@ -29,7 +29,8 @@ object ResourceHelper {
     val pipe: Option[InputStream] = try {
       val touchSource = getClass.getResourceAsStream(resource)
       Source.fromInputStream(touchSource)("UTF-8").getLines().take(1)
```

Comment:
Is not easier just to check that touchSource != null ?

Reply:
Thanks, I tried but in some cases we may get empty-iterator which is not useful either. I'll review it again but I think that won't help much.